### DISCUSSION TITLE
timeField is not a function

### DISCUSSION THREAD

**Discussion by WilkieIT** [getgauge/taiko#1806] [Development]
I have an implementation that is returning the error **timeField is not a function**

I have included it in the header

```
"use strict";
const { openBrowser,write, click, closeBrowser, goto, waitFor, focus, textBox,  intervalSecs,screenshot, into, button, dropDown, timeField } = require('taiko');
const assert = require("assert");
const headless = process.env.headless_chrome.toLowerCase() === 'true';
```
and further down I try to use it,
```
  await timeField('txtOnceDebitDate').exists();
  await write('31082020', into(timeField({id:'txtOnceDebitDate'})));
```
Any help in why this would not work would be great

**Comment by zabil** [comment]
This works on Taiko version 1.1.4. Please make sure you are on the latest version

```
const { openBrowser, goto, timeField, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto('google.com');
        await timeField('test').exists(0,0);
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();
```

If it's still an issue please share the full script, like the example above.

**Reply by WilkieIT** [reply]
you were correct.  after upgrading it worked perfectly.  Thank you. 
One suggestion I would make is that the doco has an **effective from version** field,

**Comment by NivedhaSenthil** [comment]
you can check if you have redefined the variable `timeField` in your code... like `timeField=5`

**Reply by WilkieIT** [reply]
thanks for your suggestion.  It turned out that I needed to update my version.
